src/tests/end-to-end/tests/details-view/__snapshots__/preview-features.test.ts.snap

```diff
@@ -71,7 +71,7 @@ exports[`Preview Features Panel Normal mode should match content in snapshot 1`]
             class="ms-Panel-content content-000"
           >
             <div
-              class="no-preview-feature-message"
+              class="no-preview-feature-message116bs"
```

Comment:
Consider normalizing the hash suffix in snapshots so e2e tests don't get broken if whitespace changes in a referenced CSS file; you could do this by plugging into the same helper we use to normalize the numbers in office fabric classnames in `src/tests/end-to-end/common/element-snapshot-formatter.ts` (it might be easier to distinguish the hashes if you change the `localIdentName` format to include some consistent separator for the hash, eg `'[local]---[hash:base64:5]'`)